main.py

```diff
@@ -1,7 +1,90 @@
 import init_django_orm  # noqa: F401
-
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    genres = [
+        (
+            "Western",
+        ),
+        (
+            "Action",
+        ),
+        (
+            "Dramma",
```

Comment:
There is a typo in the genre name 'Dramma'. It is later corrected to 'Drama' in the script, but ensure that this is intentional and aligns with your data requirements.